app/api/v1/views.py

```diff
@@ -88,35 +79,42 @@ def get_orders():
                                 multiple_items,
                                 start,
                                 size,
-                                "search")
+                                'search')
 
         # formatting results
         formatted_orders = SearchFunctions.format_results(orders)
         suborder_total = orders['hits']['total']
         order_total = orders['aggregations']['order_count']['value']
 
-        return jsonify(order_count=order_total,
-                       suborder_count=suborder_total,
-                       all_orders=formatted_orders), 200
+        return jsonify(
+            order_count=order_total,
+            suborder_count=suborder_total,
+            all_orders=formatted_orders,
+        ), 200
 
     else:
         orders = search_queries(date_received_start=date.today().strftime('%m/%d/%Y'))
         formatted_orders = SearchFunctions.format_results(orders)
         suborder_total = orders['hits']['total']
         order_total = orders['aggregations']['order_count']['value']
 
-        return jsonify(order_count=order_total,
-                       suborder_count=suborder_total,
-                       all_orders=formatted_orders), 200
+        return jsonify(
+            order_count=order_total,
+            suborder_count=suborder_total,
+            all_orders=formatted_orders,
+        ), 200
 
 
-@api.route('/orders/<doc_type>', methods=['GET'])
+@api.route('/orders/<string:doc_type>', methods=['GET'])
 @login_required
-def orders_doc(doc_type):
-    """
+def orders_doc(doc_type: str):
+    """Handles the export of data.
 
-    :param doc_type: document type ('csv' only)
-    :return:
+    Args:
+        doc_type: The type of document to export data into.
```

Comment:
You should probably mention valid values of doc_type in the docstring.

Reply:
Added additional info on doc_type.